### ISSUE TITLE
Trust dialog: screen reader is not very helpful

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#125112]
At least on macOS when I start VSCode with screen reader enabled and the trust dialog opens right on startup, there is no mention of a dialog in the output.

Compared to e.g. a native dialog where I get this info read out. 

I wonder how this makes it hard for users that require a screen reader to understand what is going on in VSCode when you open with trust enabled for the first time.

//cc @isidorn

**Comment by isidorn** [comment]
I already provided some accessibillty feedback here https://github.com/microsoft/vscode/issues/121094

But yes the content of the custom dialog should be read out

**Comment by sbatten** [comment]
@bpasero you are referring to the startup dialog right? (as opposed to the trust editor?) I made changes for this earlier in the milestone and I hear the dialog read on Windows with NVDA, is any part of the dialog read on macOS?

**Comment by bpasero** [comment]
@sbatten yes, the trust dialog (not the editor). I hear the button being read that has keyboard focus, but nothing else.

**Comment by sbatten** [comment]
fixed by https://github.com/microsoft/vscode/issues/127303

**Comment by connor4312** [comment]
This doesn't work for me: https://memes.peet.io/img/7fa57564-248d-4e57-8e7c-31a882021263.mp4

**Comment by sbatten** [comment]
After discussing with @connor4312, we found that this was due to live share stealing focus, will open a separate issue for them to investigate.

**Comment by sbatten** [comment]
live share issue https://github.com/MicrosoftDocs/live-share/issues/4343
